src/main/kotlin/com/terraformation/backend/customer/db/PermissionStore.kt

```diff
@@ -40,6 +42,15 @@ class PermissionStore(private val dslContext: DSLContext) {
         .fetchMap(FACILITIES.ID.asNonNullable(), ORGANIZATION_USERS.ROLE_ID.asNonNullable())
   }
 
+  /** Returns a user's funding entity */
+  fun fetchFundingEntity(userId: UserId): FundingEntityId? {
+    return dslContext
+        .select(FUNDING_ENTITY_USERS.FUNDING_ENTITY_ID)
+        .from(FUNDING_ENTITY_USERS)
+        .where(FUNDING_ENTITY_USERS.USER_ID.eq(userId))
+        .fetchOne { it[FUNDING_ENTITY_USERS.FUNDING_ENTITY_ID] }
+  }
```

Comment:
I think I'm confused as to why this is in `PermissionStore`. The similar types of lazy fetches in `IndividualUser` are all related to permissions and roles so those make sense to me to be in this class. But wouldn't we want to use a different store for this? Or is it just to reduce the number of classes that need to be added to create a `FunderUser`?

Reply:
Ah that makes sense. Because of the method name, I was thinking you were using this to retrieve the full Funding Entity for the user, but I should have looked closer to see you're just getting the id for permissions purposes